src/gsy_e/models/area/__init__.py

```diff
@@ -618,3 +620,19 @@ def update_config(self, **kwargs):
             self.strategy.read_config_event()
         for child in self.children:
             child.update_config(**kwargs)
+
+    def is_market_spot(self, market_id: str) -> bool:
```

Comment:
Nit
AFAIS the `is_market_spot`, `is_market_settlement`, `is_market_balancing` and `is_market_future` are only used in the `broadcast_notification` method.
I prefer to refrain from adding all of these, imo we can get rid of them and create 1 single method that returns the `type` of the passed market_id

Reply:
I also take advantage of these methods in many places in D3ASIM-3633, and I think that they are good to be there. The disadvantage of one single method would be that the caller code would have to also import the market type enum and compare against it in order to differentiate the market type. Therefore more imports and more verbose IMO. However it is a matter of preference IMO

Reply:
Makes sense, and I agree that it's entirely subjective.